lib/core/plugins/manager.js

```diff
@@ -1039,6 +1048,60 @@ class PluginsManager {
 
     return loadedPlugins;
   }
+
+  /**
+   * Get controller name and prefix it according to the configuration.
+   *
+   * @param {Object} plugin - Plugin Object
+   * @param {String} controller - Controller name
+   *
+   * @returns {String} - Controller name
+   */
+  _getControllerName (plugin, controller) {
+    let prefixController;
+
+    if (typeof this.config.common.prefixControllers === 'boolean') {
+      prefixController = this.config.common.prefixControllers;
+    }
+    else if (_.isArray(this.config.common.prefixControllers)) {
+      prefixController = this.config.common.prefixControllers.includes(
+        plugin.manifest.name);
+    }
+    else {
+      assert(false, 'Wrong value type for plugins.common.prefixControllers. Expect "Array" or "boolean".')
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)